readme.md

```diff
@@ -37,24 +37,20 @@ Type: `Object`
 ##### timeout
 
 Type: `number`<br>
-Default: `2000`
+Default: `5000`
 
-Milliseconds to wait for a server to respond. This option is only supported in Node.js.
+Milliseconds to wait for a server to respond.
 
-##### hostnames
+##### version
```

Comment:
When you specifically need to know if you can reach the IPv6 parts of the internet, for whatever reason. Generally, it's debatable if one can be considered 'online' when only one version works and in an ideal world, we'd do both checks, but I'd delegate this decision to the user.